src/algopy_testing/op.py

```diff
@@ -955,24 +955,242 @@ def asset_frozen(
 
 
 class _AppParamsGet:
-    def __getattr__(self, name: str) -> Any:
-        raise NotImplementedError(
-            f"AppParamsGet.{name} is currently not available as a native "
-            "`algorand-python-testing` type. Use your own preferred testing "
-            "framework of choice to mock the behaviour."
-        )
+    @staticmethod
```

Comment:
since this class is private and there is a single global instance, can make all these methods not @staticmethod, and use the self param instead